examples/api/lib/material/button_style/button_style.icon_alignment.0.dart

```diff
@@ -4,34 +4,33 @@
 
 import 'package:flutter/material.dart';
 
-// ignore: deprecated_member_use
-/// Flutter code sample for using [ButtonStyleButton.iconAlignment] parameter.
+/// Flutter code sample for using [ButtonStyle.iconAlignment] parameter.
```

Comment:
This doesn't match the code sample, either update this to like using iconAlignment parameter for various material Button.

or update the sample to passing ButtonStyle with specific iconAligment, or override the various button theme in theme data